metrics/tenant_registries.go

```diff
@@ -445,51 +463,41 @@ func sum(mf *dto.MetricFamily, fn func(*dto.Metric) float64) float64 {
 
 // max returns the maximum value from all metrics from same metric family (= series with the same metric name, but different labels)
 // Supplied function extracts value.
-func max(mf *dto.MetricFamily, fn func(*dto.Metric) float64) float64 {
-	return fold(mf, fn, func(val, res float64) float64 {
-		if val > res {
-			return val
-		}
-		return res
-	})
-}
 
 // min returns the minimum value from all metrics from same metric family (= series with the same metric name, but different labels)
 // Supplied function extracts value.
```

Comment:
Extra docstring

Reply:
Fixed in https://github.com/grafana/dskit/pull/368/commits/e3957808e55fc03b571625e853d6814cd996fa31.